Fantasy-Quest/Assets/Scripts/Interaction/Item/IInteractable.cs

```diff
@@ -2,11 +2,7 @@ namespace Interaction.Item
 {
     internal interface IInteractable
     {
-        public bool CanCatInteract
-        {
-            set => CanCatInteract = value;
-            get => CanCatInteract;
-        }
+        public bool CanCatInteract { get; }
 
         public void InteractionByCat();
```

Comment:
Make it as a verb. For example: PetCat(), Walk(), DoTransition(), Jump()

Reply:
This script is universal, there's can be any interaction(like Drink from bowl or Scratch the door). I don't know how to how to reduce the general to the specific

Reply:
I mean 'Interaction' is noun, but it should be a verb like 'Interact' according to the style code